modules/shared/src/main/scala/io/constellationnetwork/env/JarSignature.scala

```diff
@@ -0,0 +1,21 @@
+package io.constellationnetwork.env
+
+import java.io.File
+import java.nio.file.Files
+
+import io.constellationnetwork.security.hash.Hash
+
+object JarSignature {
+
+  lazy val hash: Hash = jarHash
+
+  private def jarHash = {
+    // assume fat jar
```

Comment:
We usually do not use comments over the codebase, this is necessary?

Reply:
not really, I'll  remove it.